src/main/java/org/folio/edge/orders/utils/OrdersOkapiClient.java

```diff
@@ -41,15 +41,16 @@ protected void initDefaultHeaders() {
 
   public void send(Routing routing, String payload, MultiMap params, MultiMap headers, Handler<HttpResponse<Buffer>> responseHandler,
                    Handler<Throwable> exceptionHandler) {
-
+    logger.debug("send:: Trying to send request to Okapi with routing: {}, payload: {}", routing, payload);
     final String method = routing.getProxyMethod() == null ? routing.getMethod() : routing.getProxyMethod();
 
     String resultPath = Optional.ofNullable(params)
       .map(it -> it.names().stream().reduce(routing.getProxyPath(), (acc, item) -> acc.replace(":" + item, params.get(item))))
       .orElse(routing.getProxyPath());
-
+    logger.info("send:: resultPath: {}", resultPath);
     switch (method) {
       case "POST":
+        logger.info("send:: Sending POST request to Okapi");
```

Comment:
Can we add rounting object to such logs? Is it contains something usefull in toStrign implementation?

Reply:
Ok, I will add